### ISSUE TITLE
Unicode text in slug

### ISSUE THREAD

**Issue by grapo** [ellmetha/django-machina#152]
Changes made in commit  "Dropped Django 1.11 support and switched to new-style URL definition":
from ` _(r'^forum/(?P<slug>[\w-]+)-(?P<pk>\d+)/$'),`  to ` _('forum/<slug:slug>-<int:pk>/'),` aren't equivalent for unicode slugs.
Django slug converter is ASCII only. I prefer to have ASCII slugs but in previous forum version slugs was unicode so now I have `ogólne` slug in my app and in forum version from master branch I get this error:

`django.urls.exceptions.NoReverseMatch: Reverse for 'forum' with arguments '('ogólne', 1)' not found. 1 pattern(s) tried: ['forum\\/forum\\/(?P<slug>[-a-zA-Z0-9_]+)\\-(?P<pk>[0-9]+)\\/$']`

I manage to repair this with putting this code in my app urls.py:
```
from django.urls import register_converter
from django.urls.converters import StringConverter
class SlugConverter(StringConverter):
    regex = '[\w-]+'


register_converter(SlugConverter, 'slug')
```
Overriding django slug converter is acceptable for me but it may be not for others.
IMO proper solution (that keeps backward compatibility) will be to create `machina_unicode_slug` converter, place it in django-machina urls.py and use it instead of `slug`
I can create pull request if you accept this solution.

**Comment by ellmetha** [comment]
Hey @grapo, thanks for reporting this!
I think we could simply use `str` instead of `slug` in URL path definitions too. 😉

**Comment by ellmetha** [comment]
Fixed in e500a1d. Thanks!
